spring-data-jdbc/src/main/java/org/springframework/data/jdbc/core/JdbcAggregateOperations.java

```diff
@@ -71,6 +72,17 @@ public interface JdbcAggregateOperations {
 	 */
 	<T> T insert(T instance);
 
+	/**
+	 * Inserts all aggregate instances, including all the members of each aggregate instance.
+	 *
+	 * @param instances the aggregate roots to be inserted. Must not be {@code null}.
+	 * @param <T> the type of the aggregate root.
+	 * @return the saved instances.
+	 * @throws IncorrectUpdateSemanticsDataAccessException when at least one instance is determined to be not new.
```

Comment:
Where is this coming from? If this is the case we should have a test for it. And it probably would be the same for the simple `insert` method, where we should add that documentation and the test as well.

Reply:
@schauder 
I used saveAll method of document .
That was an incorrect explanation. It's my fault.

Edited document. 
Feel free to edit it with a better description.

thank you